command/polybftmanifest/manifest_init.go

```diff
@@ -0,0 +1,211 @@
+package polybftmanifest
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/genesis"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/spf13/cobra"
+)
+
+const (
+	manifestPathFlag      = "path"
+	premineValidatorsFlag = "premine-validators"
+	validatorsFlag        = "validators"
+	validatorsPathFlag    = "validators-path"
+	validatorsPrefixFlag  = "validators-prefix"
+
+	defaultValidatorPrefixPath = "test-chain-"
+	defaultManifestPath        = "./manifest.json"
+
+	nodeIDLength       = 53
+	ecdsaAddressLength = 42
+	blsKeyLength       = 2
+)
+
+var (
+	params = &manifestInitParams{}
+)
+
+// GetCommand returns the rootchain emit command
```

Comment:
```suggestion
// GetCommand returns the polybft-manifest command
```

Reply:
even just `the command`

Reply:
Removed comment entirely, as don't find it necessary https://github.com/0xPolygon/polygon-edge/pull/1005/commits/f432f4babd18c4081d8bd25c8ade9a73badfbc28